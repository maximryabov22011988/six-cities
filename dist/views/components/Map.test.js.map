{"version":3,"file":"Map.test.js","sourceRoot":"","sources":["../../../src/views/components/Map.test.tsx"],"names":[],"mappings":";;AAAA,+BAA+B;AAC/B,gDAAgD;AAEhD,+BAAwB;AACxB,0CAAmC;AAEnC,MAAM,MAAM,GAAG,CAAC,eAAK,CAAC,CAAC;AAEvB,EAAE,CAAC,4CAA4C,EAAE,GAAG,EAAE;IACpD,aAAG,CAAC,SAAS,CAAC,iBAAiB,GAAG,IAAI,CAAC,EAAE,EAAE,CAAC;IAE5C,MAAM,IAAI,GAAG,QAAQ;SAClB,MAAM,CACL,oBAAC,aAAG,IAAC,MAAM,EAAE,MAAM,GAAI,CACxB,CAAC,MAAM,EAAE,CAAC;IAEb,MAAM,CAAC,IAAI,CAAC,CAAC,eAAe,EAAE,CAAC;AACjC,CAAC,CAAC,CAAC"}