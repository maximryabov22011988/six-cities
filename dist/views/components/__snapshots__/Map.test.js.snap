// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map renders correctly with mandatory props 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
      "top": 0,
      "zIndex": 100,
    }
  }
/>
`;
